pkg/rpcinfo/remoteinfo/remoteInfo.go

```diff
@@ -196,7 +196,29 @@ func (ri *remoteInfo) ImmutableView() rpcinfo.EndpointInfo {
 	return ri
 }
 
+func (ri *remoteInfo) getTags() map[string]string {
```

Comment:
Maybe `copyTags` is a better name?
The same the `getTagsLocks` -> `copyTagsLocks

Reply:
Why not make it a public method of the interface? This way we won't need the type assertion on line 185 (`f := from.(*remoteInfo)`)

Reply:
1. rename is done
2. currently don't have the scenario that needs to get the two maps, especially tagsLocks field